src/Files.App/Styles/ThemedIconStyles.xaml

```diff
@@ -0,0 +1,4568 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
```

Comment:
From a maintenance perspective, a XAML file with 4500 lines is not ideal 😅

Reply:
Splitting it can be done, but adding new icons would involve finding the right place to add it to.

We could use a merged dictionary for this.

Reply:
I would prefer a single file, but I'll follow your lead on this if you suggest a different approach.

Reply:
> I would prefer a single file, but I'll follow your lead on this if you suggest a different approach.

I am happy to split it up, if there is a need for smaller Xaml files - but I would like you to decide on how you would like to group them.  My Xaml regions are just how I see them being grouped, but as I wont always be the one adding new icons in the future, the grouping  should reflect your sorting @yaira2

Reply:
Would it make sense to split out the icons into a separate file and have the base template be its own file? That ways it's easier to spot where to find icons or the actual ThemeIcon control's code.

Reply:
I can split the icons into separate Xaml, the base default control template is already separate.